tests/e2e/utils/helpers.go

```diff
@@ -58,7 +59,7 @@ func GetAPMDir() string {
 }
 
 func genesisExists(subnetName string) (bool, error) {
```

Comment:
Can we add an E2E test for the chain config?

Reply:
done